app/scripts/modules/amazon/src/loadBalancer/configure/classic/AdvancedSettings.css

```diff
@@ -0,0 +1,3 @@
+.AmazonLoadBalancer-AdvancedSettings .StandardFieldLayout .sm-label-right {
+  width: 180px;
```

Comment:
shouldn't `sm-label-right` always have the same width, regardless of which dialog/section of the dialog?

Reply:
it didn't render correctly at 120px because of a long label.   This is a stopgap until we implement the design and CSS work that @gcomstock did.  Do you have a better solution?

## 120px

<img width="928" alt="screen shot 2018-11-03 at 10 06 35 am" src="https://user-images.githubusercontent.com/2053478/47955115-430b6c80-df50-11e8-8465-e337b4acbbba.png">

## 180px

<img width="918" alt="screen shot 2018-11-03 at 10 06 09 am" src="https://user-images.githubusercontent.com/2053478/47955116-4b63a780-df50-11e8-8173-aad83ca128cb.png">

Reply:
Ah, interesting. Also makes sense. 👍